Image filters

kolking opened the pull request:
Re #1526

NO FILTERS | FILTERS APPLIED
---|---
![image](https://user-images.githubusercontent.com/4656448/112038013-c9de3580-8b5b-11eb-898c-abb8aeefafb5.png) | ![image](https://user-images.githubusercontent.com/4656448/112038045-d2cf0700-8b5b-11eb-8e87-139d2f3ba9d0.png)

benwbrum commented:
This resolves #1526

benwbrum commented:
This is really fantastic!

We have one UI change to request -- can you move the controls for filtering into the OSD viewer, near the buttons for zoom, rotation, etc.?  That will be more consistent with the other image controls, and will also be visible if the user uses OSD's full-screen/pop-out option.  (It will probably require adding new buttons to match the existing OSD zoom/fullscreen/etc. buttons.)

kolking commented:
I have reworked the image toolbar and moved the image filters control into the toolbar. Now they can use the filters in fullpage mode as well.

NEW TOOLBAR | IMAGE FILTERS
---|---
![image](https://user-images.githubusercontent.com/4656448/112761131-c4289a00-900a-11eb-8859-d2102549f534.png) | ![image](https://user-images.githubusercontent.com/4656448/112761138-d0acf280-900a-11eb-858d-793bb68df241.png)
